_community_members/rishabhmaurya.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Rishabh Maurya
+short_name: rishabhmaurya
```

Comment:
@rishabhmaurya - did you also want to have a profile picture for your page? 

if so, please update - here is an example code to insert, along with adding your image file to the PR:

photo: '/assets/media/community/members/yourname.jpg'